### PR TITLE
Add reference to issue template app

### PR DESCRIPTION
**Pull request by LukasReschke**
@jospoortvliet Thoughts? We talked about that in the S-Bahn in Stuttgart on Friday :-)

If okay I'll merge this also on other repositories. But make sure the text is fine first :)

### CODE REVIEW

File: issue_template.md [hunk 1]
```diff
@@ -66,8 +67,8 @@ from within your Nextcloud installation folder
 
 or 
 
-Insert your config.php content here
-(Without the database password, passwordsalt and secret)
+Insert your config.php content here. 
+Make sure to remove all sensitive contents such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```
Comment (MariusBluem) [inline issue_template.md]: *content* is uncountable. 😁

**Review by MariusBluem — CHANGES_REQUESTED** [review]
See my comment ... Rest seems to be fine 👍

**Review by jospoortvliet — APPROVED** [review]

**Comment by jospoortvliet** [comment]
TODO: port to the other repo's. Will work on that later today.

**Comment by nickvergessen** [comment]
@jospoortvliet @MariusBluem please help review these:

### Updated issue_template.md
https://github.com/nextcloud/survey_client/pull/56
https://github.com/nextcloud/spreed/pull/280
https://github.com/nextcloud/quota_warning/pull/3
https://github.com/nextcloud/nextcloud_announcements/pull/23
https://github.com/nextcloud/files_downloadactivity/pull/14
https://github.com/nextcloud/files_automatedtagging/pull/26
https://github.com/nextcloud/files_accesscontrol/pull/60
https://github.com/nextcloud/announcementcenter/pull/80
https://github.com/nextcloud/activity/pull/134
https://github.com/nextcloud/notifications/pull/73

### New issue_template.md + contributing.md
https://github.com/nextcloud/external/pull/15
https://github.com/nextcloud/files_pdfviewer/pull/27
https://github.com/nextcloud/files_retention/pull/24
https://github.com/nextcloud/files_texteditor/pull/34
https://github.com/nextcloud/firstrunwizard/pull/38
https://github.com/nextcloud/logreader/pull/47
https://github.com/nextcloud/password_policy/pull/39
https://github.com/nextcloud/serverinfo/pull/87
https://github.com/nextcloud/templateeditor/pull/19
https://github.com/nextcloud/user_saml/pull/108

**Comment by jospoortvliet** [comment]
All reviewed and merged. I pronounce this action DONE ;-)

**Pull request merged**